app/src/main/res/menu/credit_card_editor.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/delete_credit_card_button"
+        android:icon="@drawable/ic_delete"
+        android:title="@string/credit_cards_delete_card"
+        android:visible="false"
+        app:iconTint="?primaryText"
+        app:showAsAction="ifRoom" />
+    <item
+        android:id="@+id/save_credit_card_button"
+        android:icon="@drawable/mozac_ic_check"
+        android:title="@string/save"
+        android:contentDescription="@string/save"
```

Comment:
This menu item uses a content description while the above one doesn't.
By default Talkback will read the title anyway so we can go with either approaches but for consistency one approach should be used in all  cases.